.github/sync-files.yaml

```diff
@@ -30,4 +30,4 @@
   files:
     - source: .github/workflows/build-and-test.yaml
     - source: .github/workflows/build-and-test-differential.yaml
-    - source: .github/workflows/check-build-depends.yaml
```

Comment:
Since https://github.com/autowarefoundation/autoware_launch/blob/main/build_depends.repos contains only autoware_common, I think it's unnecessary in this repository for now.